source/user-manual/api/reference.rst

```diff
@@ -6845,316 +7195,316 @@ Returns the sca checks of an agent
-	            "title": "PHP - Displaying of errors is enabled",
-	            "status": "Not applicable",
-	            "file": "/etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini",
-	            "reason": "File /etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini not found",
-	            "policy_id": "system_audit",
-	            "id": 1003,
-	            "result": "",
-	            "rules": [
-	               {
-	                  "type": "file",
-	                  "rule": "f:$php.ini -> r:^display_errors = On;"
-	               }
-	            ]
-	         },
-	         {
-	            "title": "PHP - Allow URL fopen is enabled",
-	            "status": "Not applicable",
-	            "file": "/etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini",
-	            "reason": "File /etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini not found",
-	            "policy_id": "system_audit",
-	            "id": 1002,
-	            "result": "",
-	            "rules": [
-	               {
-	                  "type": "file",
-	                  "rule": "f:$php.ini -> r:^allow_url_fopen = On;"
-	               }
-	            ]
-	         },
-	         {
-	            "title": "PHP - Expose PHP is enabled",
-	            "status": "Not applicable",
-	            "file": "/etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini",
-	            "reason": "File /etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini not found",
-	            "policy_id": "system_audit",
-	            "id": 1001,
-	            "result": "",
-	            "rules": [
-	               {
-	                  "type": "file",
-	                  "rule": "f:$php.ini -> r:^expose_php = On;"
-	               }
-	            ]
-	         },
-	         {
-	            "title": "PHP - Register globals are enabled",
-	            "status": "Not applicable",
-	            "file": "/etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini",
-	            "reason": "File /etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini not found",
-	            "policy_id": "system_audit",
-	            "id": 1000,
-	            "result": "",
-	            "rules": [
-	               {
-	                  "type": "file",
-	                  "rule": "f:$php.ini -> r:^register_globals = On;"
-	               }
-	            ]
-	         }
-	      ]
-	   }
-	}
+   {
+      "error": 0,
+      "data": {
+         "totalItems": 16,
+         "items": [
+            {
+               "title": "Web vulnerability - .htaccess file compromised - auto append",
+               "status": "Not applicable",
+               "reason": "Directory /var/www not found",
+               "policy_id": "system_audit",
+               "references": "https://blog.sucuri.net/2011/05/understanding-htaccess-attacks-part-1.html",
+               "id": 1015,
+               "result": "",
+               "compliance": [
+                  {
+                     "key": "pci_dss",
```

Comment:
This should also show the NIST 800 53 compliance.

Reply:
Fixed!